building/build-debs/homeworld-admin-tools/src/__main__.py

```diff
@@ -28,6 +29,7 @@
     "infra": infra.main_command,
     "seq": seq.main_command,
     "deploy": deploy.main_command,
+    "version": command.wrap("display version info", version.display_version)
```

Comment:
For the sake of consistency and maintaining abstraction boundaries, it makes more sense to have `version` export a `main_command` attribute that equals `command.wrap("display version info", display_version)` instead of this.

Reply:
Will change.